test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -540,7 +519,7 @@ UCS_TEST_P(test_ucp_peer_failure_keepalive, kill_receiver,
     scoped_log_handler err_handler(wrap_errors_logger);
     scoped_log_handler warn_handler(hide_warns_logger);
 
-    sendrecv(); /* allow wireup to complete */
+    smoke_test(true); /* allow wireup to complete */
```

Comment:
- pls add TODO for immediate `kill_receiver` like peer_failure does to test wireup as well
 - pls add `smoke_test` on stable pair in the end of test

Reply:
- added TODO
- added smoke test

Reply:
pls comment it's done since aux transport (ud) does not support keepalive yet